prisma/migrations/20230308235102_add_archive_table/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- CreateTable
+CREATE TABLE `archive` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `created_at` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `pull_request_link` VARCHAR(128) NOT NULL,
+    `jira_ticket` VARCHAR(128) NULL,
+    `note` TINYTEXT NULL,
```

Comment:
What is this note field used for again? Having `TINYTEXT` might be more space than we need.

Reply:
This field would store any text/note added corresponding to the PR.  Could use for reviewers' note or summary of the ticket?  Just want something simple to add some info about the PR for now.  We probably might need additional field in the future like release notes, etc...

255 chars storage.  Max string length 244 chars with 1 byte overhead. 
ref: https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html

Reply:
For now, it might be good to only store a link to the PR so that developers can easily view the PR. Later when we integrate directly with github, we can dynamically pull notes into the UI. This is not a decision we need to make in this PR, but it might be good to not store notes until we 100% need them.

Reply:
The relevant links are store on their corresponding field.  We won't be having any link add to this field.  As this note will be editable in the UI.

I do find it's helpful to have quick summary on the request without having to click on the ticket, pull request link or go to slack channel for additional info.  As for how to use this field, we can definitely decide and discuss.